src/components/molecules/Chatbox/Chatbox.scss

```
@@ -5,13 +5,18 @@
   display: flex;
   flex-flow: column;
 
-  &__messages {
+  &__messages-container {
```

```suggestion
  &__messages {
```
Any reason to change the classname?

---

reverted in https://github.com/sparkletown/sparkle/pull/1922/commits/3660c406551764e6c60917460a79a253b51c4a67